05_syntax_lab/05.py

```diff
@@ -17,6 +17,21 @@
                 print "Number %d is NOT divided by 13" % (num)
     else:
                 print "Number %d is NOT divided by 7" % (num)
+"""
+
+#New Block (fix with less indentation)

```

Comment:
We're using github :) No need to leave original block. Github shows you the diff automatically and keeps track of every past submission